### PR TITLE
doc for configuration files format

### PR DESCRIPTION
**Pull request by buixor**

### CODE REVIEW

File: docs/v1.X/docs/references/crowdsec-config.md [hunk 1]
```diff
@@ -0,0 +1,203 @@
...
+crowdsec_service:
+  #acquisition_path: ./config/acquis.yaml
+  parser_routines: 1
+cscli:
+  output: human
+  hub_branch: wip_lapi
+db_config:
+  type: sqlite
+  db_path: /var/lib/crowdsec/data/crowdsec.db
+  user: crowdsec
```
Comment (erenJag) [inline docs/v1.X/docs/references/crowdsec-config.md:30]: We should comment all parameters for MySQL and postgreSQL and let only sqlite parameters.
(event in the main `config.yaml` file)

File: docs/v1.X/docs/references/crowdsec-config.md [hunk 2]
```diff
@@ -0,0 +1,203 @@
+# Crowdsec configuration file
+
+{{v1X.crowdsec.Name}} has a main `yaml` configuration file, usually located in `/etc/crowdsec/config.yaml`.
+
+Here is an example :
+
+```yaml
```
Comment (erenJag) [inline docs/v1.X/docs/references/crowdsec-config.md:7]: If content is truncated by default, will be better.

File: docs/v1.X/docs/references/crowdsec-config.md [hunk 3]
```diff
@@ -0,0 +1,400 @@
...
+  type:     <db_type>
+  db_path:  <path_to_database_file>  
+  user:     <db_user>      # for mysql/pgsql
+  password: <db_password>  # for mysql/pgsql
+  db_name:  <db_name>      # for mysql/pgsql
+  host:     <db_host_ip>   # for mysql/pgsql
+  port:     <db_host_port> # for mysql/pgsql
+  flush:
+    max_items: <max_alerts_in_db>
+	max_age: <max_age_of_alerts_in_db>
```
Comment (buixor) [inline docs/v1.X/docs/references/crowdsec-config.md:101]: beware extra space

**Review by buixor — COMMENTED** [review]
minor remark but gj

**Review by AlteredCoder — APPROVED** [review]

**Pull request merged**